packages/cli/lib/base-generator.js

```diff
@@ -21,11 +22,113 @@ module.exports = class BaseGenerator extends Generator {
    * Subclasses can extend _setupGenerator() to set up the generator
    */
   _setupGenerator() {
+    this.option('json', {
+      type: String,
+      description: 'Stringified JSON object to configure options',
+    });
+
+    this.option('json-file', {
+      type: String,
+      description: 'JSON file to configure options',
+    });
+
+    this.option('express', {
+      type: Boolean,
+      description:
+        'Run the generator in express mode to skip optional questions',
```

Comment:
When I first saw the name of the options, I assumed `express` would be related to Express.js framework.

Can we find a better option name please, for example `--non-interactive` or `--skip-prompts`?